debian/control

```diff
@@ -22,12 +22,12 @@ Build-Depends: debhelper (>= 10),
                libfuse3-dev,
                libprotobuf-c-dev,
                libjson-c-dev,
-               dpdk-dev (>= 21.11.1),
+               dpdk-dev (>= 21.11.2),
```

Comment:
Now that I look at this more closely, this appears to deviate from the RPM's dependencies in that `daos.spec` does not have a dependency on `dpdk-devel` because that's the job of `spdk-devel`.  I suspect that the debian spdk-dev has the same kind of dependency, and if it doesn't it should.  Either way this dependency should be removed from here.

But given the number of iterations on this PR already, I guess we can let this slide.  But please try to remember to do this on the next spdk/dpdk update?  Maybe commit it to your local tree(s) to remind you to include it in your next set of PR(s)?

Reply:
I will address in my next PR

Reply:
Just checked and `spdk-dev` does have the `dpdk-dev (>= 21.11.2)` `BuildDepends`, so if you have to push to this PR for any other review concerns, can you please remove this `dpdk-dev (>= 21.11.2)` `BuildDepends` also?